UtopiaCity/Controllers/Timeline/BreadcrumbActionFilterAttribute.cs

```diff
@@ -0,0 +1,142 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text.RegularExpressions;
+using UtopiaCity.Models.TimelineModel.CollectionDataModel;
+
+namespace UtopiaCity.Controllers.Timeline
+{
+    public class BreadcrumbActionFilterAttribute : ActionFilterAttribute
```

Comment:
I thought there was a nuget package for "BreadCrumb" functionality, and no reason to implement this by your own, but maybe i'm wrong and you have to do this , anyway looks interesting